docs/en/ingest-management/overview.asciidoc

```diff
@@ -143,10 +143,12 @@ ships them to the {agent}s. To communicate to {fleet} about the status of the
 == {agent} self-protection
 
 On macOS and Windows, {agent} can self-protect against malicious users and
-attackers when the {endpoint-sec} integration is added to the agent policy. For
+attackers when the {endpoint-cloud-sec} integration is added to the agent policy. For
 more information, refer to
 {security-guide}/es-overview.html#self-protection[{elastic-endpoint} self-protection].
 
+//REVIEWERS: Is Elastic Endpoint correct here? ^^
```

Comment:
This looks correct. Self protection is implemented in the Endpoint executable. When Endpoint and Cloud Security integration is added to Agent, Endpoint will take measures to protect itself and Agent on macOS and Windows.

I'm not sure how much we need or want to document _which_ process implements the self protection, whether to say "Agent" or "Endpoint" here seems to be a matter of style that probably ought to match how things are phrased elsewhere (for instance would we say "When the Endpoint Security integration is added to an Agent, Agent can prevent malware from executing on the host" or "When the Endpoint Security integration is added to an Agent, that integration can prevent malware from executing on the host"?